enterprise/tools/rbesxs/rbesxs.go

```diff
@@ -0,0 +1,222 @@
+// Utility for running a remote execution side-by-side test.
```

Comment:
I wasn't sure what side-by-side meant initially - maybe worth adding a sentence or two more on what this tool does

Reply:
Good call, updated the comment.